examples/tests/invalid/capture_mutables.w

```diff
@@ -2,11 +2,15 @@ let a = MutArray<str>["hello"];
 let s = MutSet<num>{12};
 let m = MutMap<bool>{"hello": true};
 
+let a_cloned = (Array<str>["hello"]).copy_mut();
+
 let i = inflight () => {
   assert(a.length == 1);
   //     ^ Cannot reference 'a' of type 'MutArray<str>' from an inflight context
   assert(s.size == 1);
   //     ^ Cannot reference 's' of type 'MutSet<num>' from an inflight context
   assert(m.size == 1);
   //     ^ Cannot reference 'm' of type 'MutMap<bool>' from an inflight context
+  assert(a_cloned.size == 1);
+  //     ^ Cannot reference 'a_cloned' of type 'MutArray<str>' from an inflight context
```

Comment:
Maybe we can tweak the messaging to make it clearer why it can't be referenced (we mention the type but it won't be obvious to all users that the type being mutable is the issue)

"Cannot reference mutable value 'a_cloned' of type 'MutArray<str>' from an inflight context"